### DISCUSSION TITLE
YT Playlist Download and Merge into MP4?

### DISCUSSION THREAD

**Discussion by aesculusglabra** [jely2002/youtube-dl-gui#320] [Q&A] [answered]
Hello, I'm getting an error when I try to download a particular playlist I created.

https://www.youtube.com/playlist?list=PLB6asNE3ghTbQ6gOkwyFy27ff4YdcfZLJ

I was hoping to download and merge (at the same time), if that's possible? Also, I noticed some of the individual files download as MP4 and some download as MKV. Why is that?

Thanks

**Comment by aesculusglabra** [comment, marked as answer]
The application FFWORKS performs this perfectly. Using Video/Audio PassThru for no loss in quality.

**Comment by jely2002** [comment]
Yep, the app itself can't do this. It's something for another app,  this is a downloader, not a video editor per se.
